cefi/handler/capitalcom.py

```diff
@@ -80,6 +81,7 @@ async def get_quote(self, instrument):
         :return: The fetched quote.
         """
         try:
+            logger.debug("Client: {}", self.client)
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 suggestion (security):** Consider the sensitivity of logging the entire client object.

Logging the entire client object might inadvertently expose sensitive information. It might be more prudent to log specific, non-sensitive details.

```suggestion
logger.debug("Client ID: {}", self.client.id)
```